tests/integration/security/filebased_tls_origination/main_test.go

```diff
@@ -35,7 +35,7 @@ func TestMain(m *testing.M) {
 		NewSuite(m).
 		Label(label.CustomSetup).
 		Label("CustomSetup").
-		SkipExternalControlPlaneTopology().
+		RequireMultiPrimary(). // SkipExternalControlPlaneTopology(). TODO this test fails when running with istiodless remotes
```

Comment:
This is reversing a recent change in #39156. Seems it doesn't work after switching to istiodless remotes?
^^^ @aryan16

Reply:
That's strange, let's merge this PR and I will try to take a closer look on this after that.